src/io/linux.zig

```diff
@@ -109,12 +120,6 @@ pub const IO = struct {
             self.completed = .{};
             while (copy.pop()) |completion| completion.complete();
         }
-        // Again, loop on a copy of the list to avoid an infinite loop:
```

Comment:
What do we expect here as we exit? Should there be anything in `unqueued`? If not, let's assert that. Anything surprising that might delay scheduling.

Reply:
It would be good to take a quick look also through the other control plane functions here in the vicinity and see if we can assert for surprising scheduling bugs. This may surface some things.

Reply:
Here, `unqueued` can still have entries either by 1) those who didn't get an SQE when popping unqueued above or 2) completed callbacks which enqueue() and don't get an SQE, same condition. It could also be empty if both cases don't manage to overflow the SQE array.

Should there be a comment here explaining this?

Reply:
Yes, if we don't have an assert, it would be great to explain why.